templates/admin/controls/search-settings.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @var \Swiftype\SiteSearch\Wordpress\Admin\Page $this
+ */
+
+$engineSlug   = $this->getConfig()->getEngineSlug();
+$documentType = $this->getConfig()->getDocumentType();
+
+$searchSettingsRows = [
+  [
+    'title'        => __('Weights'),
+    'description'  => __("Your search relevance function determines the order of your search results."),
+    'url'          => sprintf("https://app.swiftype.com/engines/%s/document_types/%s/custom_queries/", $engineSlug, $documentType),
+    'button_title' => __("Manage Weights"),
+  ],
+  [
+    'title'        => __('Result rankings'),
+    'description'  => __("Search your engine and re-order the results to your liking."),
+    'url'          => sprintf("https://app.swiftype.com/engines/%s/document_types/%s/search/", $engineSlug, $documentType),
+    'button_title' => __("Manage Rankings"),
+  ],
+  [
+    'title'        => __('Synonyms'),
+    'description'  => __("Synonyms are groups of terms that will be treated as equivalent for the purposes of search."),
```

Comment:
It looks like these are repeated in the UI.

<img width="1031" alt="repeat" src="https://user-images.githubusercontent.com/1427475/53434510-ce5bb280-39c4-11e9-9d73-620d26f2a974.png">

Reply:
You are right. The column was duplicated into the template.
Fixed by 604b0e8. Let you test again and resolve the conversation.